images/capi/ansible/roles/setup/tasks/main.yml

```
@@ -16,7 +16,10 @@
   when: ansible_os_family == "Debian"
 
 - import_tasks: flatcar.yml
-  when: ansible_os_family == "Flatcar"
+  # This task will override ansible_os_family to "Flatcar" as a workaround for
```

```suggestion
  # This task overrides ansible_os_family to "Flatcar" as a workaround for
```

Avoid future tense in docs.